.travis.yml

```diff
@@ -107,16 +107,15 @@ script:
     travis_retry docker exec -it pycbc_inst /bin/bash -lc 'df -h; exit $?' || exit 1 ;
     travis_retry docker exec -it pycbc_inst /bin/bash -lc 'cd /pycbc; pip install -r requirements.txt ; exit $?' || exit 1 ;
     travis_retry docker exec -it pycbc_inst /bin/bash -lc 'cd /pycbc; pip install . ; exit $?' || exit 1 ;
-    travis_retry docker exec -it pycbc_inst mkdir -p /opt/pycbc/src || exit 1;
-    travis_retry docker exec -it pycbc_inst /bin/bash -lc 'cd /opt/pycbc/src; git clone https://github.com/ligo-cbc/pycbc.git ; exit $?' || exit 1 ;
     if [ "x${TRAVIS_SECURE_ENV_VARS}" == "xtrue" ] ; then
-      LAL_EXTRA_TAG="e02dab8c" ;
+      LAL_EXTRA_TAG="current" ;
       travis_retry docker exec -it pycbc_inst mkdir -p /opt/pycbc/pycbc-software/share/lal-data || exit 1 ;
```

Comment:
Any way we can remove most of the travis commands (or all) from .travis.yml. Some of these at least probably should just be in the dockerfile to begin with such as the lal-data stuff, no?

Reply:
@ahnitz I moved these to a script inside the Docker image. The way Travis works is that you need to mount the Travis pwd inside the container and install that.